erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -250,7 +252,7 @@ exports.ErizoJSController = (erizoJSId, threadPool, ioThreadPool) => {
     updateUptimeInfo();
     let publisher;
     log.info('addPublisher, clientId', clientId, 'streamId', streamId);
-    const client = getOrCreateClient(erizoControllerId, clientId, options.singlePC);
+    const client = getOrCreateClient(erizoControllerId, clientId, options, options.singlePC);
```

Comment:
I assume we do this because we want to have the options passed when adding the publisher. I think, however, that we should still use that options structure and not pass `options.singlePC`. We can do whatever check we need in `getOrCreateClient` to avoid any problems.